src/App.js

```diff
@@ -1,13 +1,28 @@
 import React, { Component } from 'react';
 import { hot } from 'react-hot-loader';
+import { BrowserRouter as Router, Route, Link } from 'react-router-dom';
+import LandingPage from './views/LandingPage/LandingPage';
+import LoginPage from './views/LoginPage/LoginPage';
 
 class App extends Component {
   render() {
     return (
-      <div>
-        <h1>Team Persephone - Author's Haven</h1>
-      </div>
+      <Router>
+        <ul>
```

Comment:
Great Job Tunde, I noticed you added some list elements on the app component, I'm a bit unclear about why you added them. Can you enlighten me on why you chose to add them?

Reply:
Thanks, @fxola, regarding the list elements, I added them to demonstrate the use case of the react-router. they are acting as the link to the demo pages I created. hope this is clear enough?

Reply:
Oh Cool 👍🏼